lib/utils/resolveSilent.mjs

```diff
@@ -0,0 +1,46 @@
+import fs from 'node:fs';
+import path from 'node:path';
+
+import { fileURLToPath, pathToFileURL } from 'node:url';
+
+import { moduleResolve } from '@dual-bundle/import-meta-resolve';
```

Comment:
[ask] Why does it use `moduleResolve` instead of [`resolve`](https://github.com/wooorm/import-meta-resolve#resolvespecifier-parent)? `resolve` seems closer to the native `import.meta.resolve`.

Reply:
`resolve` does not apply [Resolver algorithm](https://nodejs.org/dist/latest-v14.x/docs/api/esm.html#esm_resolver_algorithm), so `moduleResolve` is slower than `resolve`.

Reply:
I see. Thanks for the link 👍🏼